src/panels/config/ha-panel-config.ts

```diff
@@ -57,22 +57,22 @@ export const configSections: { [name: string]: PageNavigation[] } = {
     {
       path: "/config/automation",
       name: "Automations & Scenes",
-      description: "Automations, blueprints, scenes and scripts",
+      description: "Automations, helpers, scenes and scripts",
```

Comment:
Shouldn't helpers be the last item?

Reply:
I don’t know? Should it?

Reply:
Yeah, lets change it to `Automations, scenes, scripts and helpers`

Reply:
What would the rationale between the current change and the discussed one?

Reply:
And what is that?
The rationale behind this PR is that @balloob told me to change to this earlier today

Reply:
I was not referring the the PR as a whole, merely this suggestion. The discussion in this review comment is about the order, but there is no explanation why the order of the items in that description should be the one or the other order.

For example: "I think helpers should be placed at the end, because the sun is shining"

Reply:
Automations is the most used menu item in this list by my assumption. Scenes is logic next to that, because we always talk about Automations and Scenes. Scripts and Helpers both support the first two and therefore should be after these. I don't have a preference if its should be `Automations, scenes, scripts and helpers` or `Automations, scenes, helpers and scripts`. I suggested the first one because that's the current order.

Reply:
> Automations is the most used menu item in this list by my assumption
Agreed.

> Scenes is logic next to that, because we always talk about Automations and Scenes

Actually, that I disagree on. I think scenes in general are lesser used. Furthermore, we generally talk about "Automation & Scripts" a lot (that is almost in every release notes even), because they are closely related.

I dare to say that, most people use "Scripts" as a "Scenes" replacement, as it is more flexible and generally easier for most. Unfortunately, we cannot conclude that from our analytics; as we don't track scenes and scripts.

In terms of looking at what belongs together, both scenes & helpers are used in automations & scripts. So grouping those together might make sense as well.

Lastly, just sort it alphabetically... satisfies the OCD brains :)